### PR TITLE
Add a generic error page for non-404 errors

### PR DESCRIPTION
**Pull request by Vinnl**
# References: 
Jira: MNTOR-1201
Figma: https://www.figma.com/file/gvyyqS9egsGjsH0LUDqT1m/Breach-Resolution-Flow-Design?node-id=960%3A19416&t=RFh8taWN70B5Ct7q-4

# Description

This adds an error page for non-404 headers. **The only thing I haven't figured out is why the user is logged out when encountering this - I'm not sure which layer maintains the user session?**

# Screenshot (if applicable)

image

# How to test

Sign in with a non-admin user, then visit `/admin/`.

# Checklist (Definition of Done)
- [x] Localization strings (if needed) have been added.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [x] I've added or updated the relevant sections in readme and/or code comments
- [ ] I've added a unit test to test for potential regressions of this bug.
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [x] All acceptance criteria are met.
- [x] Jira ticket has been updated (if needed) to match changes made during the development process.
- [x] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

**Review by flodolo — APPROVED** [review]

File: src/client/css/partials/error.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.error-page {
```
Comment (flozia) [inline src/client/css/partials/error.css:1]: Do we also need the license header for stylesheets?

File: src/client/css/partials/error.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  gap: var(--padding-sm);
+}
+
+.error-page .error-code {
+  font: var(--h3-font);
+  font-weight: bold;
+  font-size: 10rem;
+  color: var(--purple-60);
+  text-shadow: 0.75rem 0 0 var(--purple-20);
+}
```
Comment (flozia) [inline src/client/css/partials/error.css:24]: Empty line missing.

**Review by flozia — APPROVED** [review]
Looking good!

File: src/client/css/partials/error.css (hunk 1 above)
Comment (Vinnl) [reply src/client/css/partials/error.css:1]: I just spent too much time trying to set up a linting rule for this, but something in our linting/VSCode setup keeps making it remove arbitrary lines and it's getting annoying, so I gave up. In any case, most CSS files don't have it, and AFAIK the licence doesn't require it, so I'm giving up at this point.

File: src/client/css/partials/error.css (hunk 2 above)
Comment (Vinnl) [reply src/client/css/partials/error.css]: Here too, somehow it got automatically removed, and now it no longer does it... Fixed in 8199089a, but I've got to figure out what's going wrong here sometime :/

**Pull request merged**